build/install_requirements.sh

```diff
@@ -63,9 +67,12 @@ then
   sudo cp raspberrypi.pip.conf /etc/pip.conf
 fi
 
+
+# install cython v0.29 to address a compile time bug for h5py
+# see https://github.com/h5py/h5py/issues/1533#issuecomment-617519250
+sudo pip3 install cython
```

Comment:
Do we really need cython for picamera? I've used the picam lib for a number of projects without cython.

Reply:
h5py is also available as a binary from the rpi binary repo. We've been downloading it without issue. No need to build from source.

Reply:
No indeed, it has fallen in as a cascade of bug I created!